puppet/modules/slave/manifests/init.pp

```diff
@@ -162,6 +162,19 @@
     }
   }
 
+  # Needed by foreman_openscap gem dependency OpenSCAP
+  if $osfamily == 'RedHat' {
+    yumrepo { 'isimluk-openscap':
+      enabled  => 1,
+      gpgcheck => 0,
+      baseurl  => "http://copr-be.cloud.fedoraproject.org/results/isimluk/OpenSCAP/epel-${::operatingsystemmajrelease}-\$basearch/",
+      includepkgs => ['openscap'],
+    } ->
+    package { 'openscap':
```

Comment:
should this be openscap-devel if you're compiling against it (e.g. https://github.com/theforeman/foreman-packaging/blob/rpm/develop/rubygem-openscap/rubygem-openscap.spec#L35)?  I assume you're adding this in order to build the openscap gem?